development-guild/contributions/fetch-users.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs');
+const { stringify } = require('csv-stringify');
+
+const GRAPHQL_URL = 'https://coordinape-prod.hasura.app/v1/graphql';
+console.log('GRAPHQL_URL:', GRAPHQL_URL);
+
+async function fetchUsers() {
+    console.log('fetchUsers');
+
+    const response = await fetch(GRAPHQL_URL, {
+        method: 'POST',
+        headers: {
+            'Authorization': 'Bearer api|E0WXWOBQRpU4S1DTXSiCSkKEcZJ09m0sM7QKwETu',
```

Comment:
And here, can this be in config?

Reply:
@johnmark13 Yes, fixed:  https://github.com/nation3/citizen-rewards/pull/6/commits/e7dd45d18b26f6aa45511f1302042793f9fbc14f